### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by MDzola**
just to make sure it works

### CODE REVIEW

**Review by ThomasGreene — APPROVED** [review]

File: scripts/api_Restaurant.js [hunk 1]
```diff
@@ -43,12 +43,12 @@ function AddFoodComponentToDom(nameOfRestaurant) {
...
     return `
         <h2 id = rest-${uniqueRestId}>${foodObj.restaurant.name}</h2>
-            <button id = butt-${uniqueButtonId}>Save</button>
+            <button id = butt-${uniqueRestButtonId}>Save</button>
     `;
 }
 
 function putMyRestaurantIntoTheItinerary (restaurantElement) {
-    document.getElementById("restaurant-itinerary").innerHTML = restaurantElement;
+    document.getElementById("restaurant-itinerary").innerHTML = "Restaurant: " + restaurantElement;
```
Comment (lindseyemaddox) [inline scripts/api_Restaurant.js:51]: Yeah we just need to keep an eye on how this puppy is going to behave vs. the others. And we need to remember to inspect to make sure the HTML structures are identical so we don't end up with styling anomolies!

**Review by lindseyemaddox — APPROVED** [review]

File: scripts/api_Restaurant.js (hunk 1 above)
Comment (MDzola) [reply scripts/api_Restaurant.js:51]: i gotcha, I wasnt sure at first what you were getting at cause im crap at styling but i think i understand now!

Comment (MDzola) [reply scripts/api_Restaurant.js:51]: let me merge and if you want to take that part out for a smoother styling you have my blessing, I just want to make sure its working for everyone first

**Review by pricehome — APPROVED** [review]
OK

**Pull request merged**